### PR TITLE
config: adds literature categories

### PR DESCRIPTION
**Pull request by jacenkow**
* Adds translations of Inspire literature categories,
  available at app.config['LITERATURE_CATEGORIES'].

Signed-off-by: Grzegorz Jacenków <grzegorz.jacenkow@cern.ch>

### CODE REVIEW

**Comment by jacenkow** [comment]
@kaplun

File: inspirehep/config.py [hunk 1]
```diff
@@ -672,3 +672,25 @@ def _(x):
 # Submission
 # ==========
 CFG_ROBOTUPLOAD_SUBMISSION_BASEURL = "http://localhost:5000"
+
+# Inspire subject translation
+# ===========================
+LITERATURE_CATEGORIES = {
```
Comment (kaplun) [inline inspirehep/config.py]: This is actually a subset of: https://github.com/inspirehep/inspire/blob/master/bibconvert/arxiv-to-inspire-categories.kb

Can you use that source?

Comment (kaplun) [inline inspirehep/config.py]: Could be called ARXIV_TO_INSPIRE_CATEGORY_MAPPING

Comment (kaplun) [inline inspirehep/config.py]: @Panos512 I think this list could be useful to you too :)

Comment (jacenkow) [inline inspirehep/config.py]: Yeah no worries 🤓

Sent from my banana

**Comment by jacenkow** [comment]
@kaplun done.

**Pull request merged**